resolver/oneInchmulti.js

```diff
@@ -0,0 +1,27 @@
+import axios from "axios";
+export async function ExecuteQuote(quoteParams) {
+  for (var i = 0; i < quoteParams.fromTokenAddressArray.length; i++) {
+    var singlequoteParams = {
+      fromTokenAddress: quoteParams.fromTokenAddressArray[i],
+      toTokenAddress: quoteParams.toTokenAddressArray[i],
+      amount: quoteParams.amountArray[i],
+    };
+    const chainId = 56; //For Binance smart chain
+    const apiBaseUrl = "https://api.1inch.io/v4.0/" + chainId;
+    const url =
+      apiBaseUrl +
+      "/quote" +
+      "?" +
+      new URLSearchParams(singlequoteParams).toString();
+    await axios
+      .get(url)
```

Comment:
Pass params like this, do not concat them with the URL.
```suggestion
    await axios
      .get(url, {params: params})
```

Reply:
Okay. Made the change